app/src/org/commcare/android/tasks/ExternalManageKeyRecordTask.java

```diff
@@ -0,0 +1,57 @@
+package org.commcare.android.tasks;
+
+import android.content.Context;
+
+import org.commcare.android.models.notifications.MessageTag;
+import org.commcare.android.models.notifications.NotificationMessage;
+import org.commcare.dalvik.activities.DataPullController;
+import org.commcare.dalvik.activities.LoginMode;
+import org.commcare.dalvik.application.CommCareApp;
+
+/**
+ * @author Phillip Mates (pmates@dimagi.com).
+ */
+public class ExternalManageKeyRecordTask extends ManageKeyRecordTask<ExternalManageKeyRecordTask.DummyDataPullController> {
+    public ExternalManageKeyRecordTask(Context c, int taskId, String username, String password,
+                                       LoginMode loginMode, CommCareApp app, boolean restoreSession) {
+        super(c, taskId, username, password, loginMode, app, restoreSession, false);
+    }
+
```

Comment:
would be worth adding an explanatory comment for why this class overrides all of its inherited methods with empty implementations. Or just rename it DummyManageKeyRecordTask (assuming this is a Dummy class?)